server/api/api/endpoints/model_endpoints.py

```diff
@@ -461,11 +475,36 @@ async def get_model_endpoint_monitoring_metrics_values(
 
     :returns:      A list of the results values for this model endpoint.
     """
-    return await run_in_threadpool(
+
+    read_data_coroutine = run_in_threadpool(
         mlrun.model_monitoring.db.v3io_tsdb_reader.read_data,
         project=params.project,
         endpoint_id=params.endpoint_id,
         metrics=params.metrics,
         start=params.start,
         end=params.end,
     )
+
+    if any(
+        [
+            metric.name
+            == mlrun.common.schemas.model_monitoring.ModelMonitoringStoreKinds.EVENTS
```

Comment:
It's not the proper condition and not the correct name... It must match the returned FQN.